code-challenges.test.js

```diff
@@ -55,6 +99,35 @@ var numbersToAdd2 = [0, 7, -8, 12]
 var numbersToAdd3 = []
 // Expected output: []
 
+describe('increaseSum', ()=>{
+  it('returns an array of an accumulating sum', ()=>{
+    expect(increaseSum(numbersToAdd1)).toEqual([2, 6, 51, 60])
+    expect(increaseSum(numberToAdd2)).toEqual([0, 7, -1, 11])
+  })
+})
 
+// GOT RED
 
 // b) Create the function that makes the test pass.
+
+// create a function that takes in an array as an arguement
+// iterates through the existing array using .map()
+// adds the previous value in the array to the next value
+// returns an array (same length) of an accumulated sum
+
+// I also got stuck here using my original thought process^ I wasn't successful so I tried researching to find a solution
+// like the function written below, but I still wasn't able to get the output I wanted
```

Comment:
Great job trying to get the reduce function to work! It's definitely complicated